docs/storage.md

```diff
@@ -70,6 +75,8 @@ If your local storage becomes corrupted for whatever reason, your best bet is to
 
 If both time and size retention policies are specified, whichever policy triggers first will be used at that instant.
 
+Expired block cleanup happens on a background schedule, so it may take some time (hours) after startup to remove expired blocks. Expired blocks must be fully time expired before they are cleaned up.
```

Comment:
fully time expired reads oddly.

It'll happen on the next reload() ,which is usually every 2h.

Reply:
Yea, I wasn't happy with that wording either. Will fix it up.